sections/introduction.tex

```diff
@@ -0,0 +1,64 @@
+\section{Introduction}\label{sec:introduction}
+% Intro to recommendation and techniques
+News is being broadcast every day around the world in the form of news articles, television, and newspapers, which supply people with the latest information.
+Searching and categorizing the news is becoming a bigger problem since news is created at all times.
+As the internet becomes more personalized, as seen with targeted advertisements, recommender systems are seeing more use in applications, such as Netflix and Amazon.
+Within recommendation, there are two main machine learning techniques: collaborative filtering and content-based filtering.
```

Comment:
```suggestion
Searching and categorizing the news is becoming a bigger problem since news is created at all times. 
Topic modeling is an approach that takes a set of documents and generates topics that can be used for categorizing text.
```